alacritty/src/config/test.rs

```diff
@@ -1,24 +0,0 @@
-use alacritty_terminal::config::DEFAULT_ALACRITTY_CONFIG;
-
-use crate::config::Config;
-
-#[test]
-fn parse_config() {
-    let config: Config =
-        ::serde_yaml::from_str(DEFAULT_ALACRITTY_CONFIG).expect("deserialize config");
-
-    // Sanity check that mouse bindings are being parsed
-    assert!(!config.ui_config.mouse_bindings.is_empty());
-
-    // Sanity check that key bindings are being parsed
-    assert!(!config.ui_config.key_bindings.is_empty());
-}
-
-#[test]
-fn default_match_empty() {
-    let default = Config::default();
-
-    let empty = serde_yaml::from_str("key: val\n").unwrap();
-
-    assert_eq!(default, empty);
-}
```

Comment:
We don't need a separate `test.rs` file, but the `EOF` error is something that occurs when trying to load the default configuration file, right?

So it would probably make sense to add just one simple test that would inform us when our EOF error message checking has failed. Just a simple deserialization of the documentation file and checking that it matches the default.